### ISSUE TITLE
Obsolete outer SimInfoManager

### ISSUE THREAD

**Issue by rubenbaetens** [open-ideas/IDEAS#466]
I notice in (e.g.) `IDEAS.Buildings.Components.Window` that

- IDEAS.Buildings.Components.BaseClasses.ExteriorConvection
- IDEAS.Buildings.Components.BaseClasses.HeatRadiation
- IDEAS.Climate.Meteo.Solar.RadSolData
- ...

no longer use the `SimInfoManager` and thus the `outer SimInfoManager` seems obsolete here.

(allthough i liked the solution with the outer SimInfoManager much more compared to the `weaBus` as it is now, i.e. sometimes `weaBus`, sometimes `RealInput`)

**Comment by Mathadon** [comment]
I agree that the current implementation is quite messy and improvement is possible. However, we need the weaBus for our research so I'd like to stick with that. Also, connecting a weaBus to even the smallest of components seems a bit messy.. 
I'd prefer not to change it since it works now..

**Comment by rubenbaetens** [comment]
What's the real need for `weaBus` in research ?

**Comment by Mathadon** [comment]
Linearisation possibilities!

**Comment by Mathadon** [comment]
I'll try to clean it up a bit since I'm also experiencing some problems with this.

**Comment by Mathadon** [comment]
In `Window` we still have:
```
  Climate.Meteo.Solar.RadSolData radSolData(
    inc=inc,
    azi=azi,
    numAzi=sim.numAzi,
    offsetAzi=sim.offsetAzi,
    ceilingInc=sim.ceilingInc,
    lat=sim.lat)
``` 
so `sim` should stay for that reason too :)

**Comment by rubenbaetens** [comment]
Addressed in #472.
